src/ontology/mondo-edit.obo

```diff
@@ -444446,16 +444418,29 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0020772
 name: epilepsy, juvenile absence, susceptibility to, 1
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: rare
```

Comment:
subset should not be added manually. These will be added automatically via the update with orphanet, NORD, GARD.

Reply:
So all of these 5 subsets should be removed?

Reply:
yes